### PR TITLE
Stop using a named helm repo for cert-manager, link directly

### LINKED ISSUE #1100: Minor refactor to reduce helm emitted log noise and increase performance
In https://github.com/2i2c-org/infrastructure/pull/1091#pullrequestreview-909843910 I made a code suggestion I want to get done after #1091 is merged in a dedicated PR. This is an issue to represent the need to get that done.

### PR DESCRIPTION
**Pull request by consideRatio**
Closes #1100

### CODE REVIEW

File: deployer/cluster.py [hunk 1]
```diff
@@ -73,36 +73,17 @@ def deploy_support(self):
...
-            ]
-        )
-
         print_colour("Provisioning cert-manager...")
         subprocess.check_call(
             [
                 "helm",
                 "upgrade",
+                "cert-manager",  # given release name (aka. installation name)
+                "cert-manager",  # helm chart to install
```
Comment (yuvipanda) [inline deployer/cluster.py:82]: I thought if we don't explicitly add the repo and specify where it comes from, it comes from the old deprecated kubernetes/charts repo, and we'll get https://github.com/helm/charts/tree/master/stable/cert-manager

Comment (consideRatio) [reply deployer/cluster.py:82]: The line below, `f"--repo={cert_manager_url}",`, is added to explicitly make us rely on the helm chart repo specified with `--repo`, and search for a chart named `cert-manager` in that.

Comment (yuvipanda) [reply deployer/cluster.py:82]: I just somehow completely missed that!

**Review by yuvipanda — APPROVED** [review]

**Review by choldgraf — APPROVED** [review]
image
YesGuyYesManGIF

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -44,4 +44,4 @@ repos:
     hooks:
       - id: sops-encryption
         # Add files here if they contain the word 'secret' but should not be encrypted
-        exclude: secrets\.md
+        exclude: secrets\.md|helm-charts/support/templates/prometheus-ingres-auth/secret\.yaml
```
Comment (sgibson91) [inline .pre-commit-config.yaml:47]: Does this have to be a list of specific files, or can we make it "any file under helm-charts/**/templates/** that has secret in it's name"? Like in our git-ignore?

https://github.com/2i2c-org/infrastructure/blob/master/.gitignore#L16

Comment (consideRatio) [reply .pre-commit-config.yaml:47]: This must be a python regular expression i think, so i dont think so :/

Comment (sgibson91) [reply .pre-commit-config.yaml:47]: It's not critical, just a wondering :)

**Comment by yuvipanda** [comment]
Gonna merge to get CI happy again, and I think this is an easy enough one.

**Comment by yuvipanda** [comment]
Thanks a lot, @consideRatio!

**Pull request merged**